website/code_issues.qmd

```diff
@@ -227,4 +227,34 @@ The issue described in the recipe, also applies to the use of other function whi
 
 --------------------------
 
+# Writing Files and Directories to the Home Filespace
+
+## Problem
+
+You are writing by default or in your examples, vignettes or tests in the user's home filespace, including the package directory and `getwd()`.
+
+## Solution 
+
+Omit any default path in your functions and write to `tempdir()` in examples, vignettes or tests.  
+
+
+### Details
+
+<!--adding this is a test for including the CRAN mail "template" for each recipe. This version is a small collapsible text box which I will add for (almost) every Issue. Maybe change the title of the boxes?--> 
+
+::: {.callout-note title="Comment of CRAN maintainers" appearance="simple" icon=false collapse=true}
```

Comment:
I think the collapsible feature is good here. Perhaps we could change the title of the callout? (since they will be with nearly every recipe), here are some ideas 💡 :

- Comments from CRAN Maintainers
- CRAN Review Communication
- CRAN Review Feedback

Reply:
Yeah, makes sense. I also wasn't quite happy with the title. I think, I like "CRAN Review Communication" best. Maybe we come up with an even better title in the future but for now let's stick to that.